rust/src/address.rs

```diff
@@ -36,6 +36,7 @@ fn variable_nat_encode(mut num: u64) -> Vec<u8> {
 #[derive(Debug, Clone, Eq, Hash, Ord, PartialEq, PartialOrd)]
 pub struct NetworkInfo {
     network_id: u8,
+    // FIXME: should by i32?
```

Comment:
This comment is wrong. I took this from the cardano-rust codebase, but in the CDDL for both Byron & Shelley it's an u32